src/pages/Parkmoonyoung/Login/LoginPmy.js

```diff
@@ -1,6 +1,77 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './LoginPmy.scss';
+import { Link } from 'react-router-dom';
 
 export default function LoginPmy() {
-  return <div>Login_pmy</div>;
+  return (
+    <div className="loginWrap">
+      <div className="loginArea">
+        {/* <h1 className="logo"><a href="">instagram</a></h1> */}
+        <HeadingTag1 />
+
+        <InputDataArea />
+
+        <Link to="/main" className="forgotPw">
+          비밀번호를 잊으셨나요?
+        </Link>
+      </div>
+    </div>
+  );
+}
+
+function HeadingTag1() {
+  return (
+    <h1 className="logo">
+      <Link to="/signup">westagram</Link>
+    </h1>
+  );
+}
+
+function InputDataArea() {
+  let [isDisabled, setDisabled] = useState('disabled');
+  let [isIdValue, setIdValue] = useState('');
+  let [isPwValue, setPwValue] = useState('');
+
+  function saveUserId(e) {
+    setIdValue(e.target.value);
+    return isIdValue;
+  }
+
+  function saveUserPw(e) {
+    setPwValue(e.target.value);
+    return isPwValue;
+  }
```

Comment:
- 변수는 특이사항이 없다면 const로 선언해주세요
- e.target.value를 받아 state변수에 넣는 로직이 동일하므로 하나로 합쳐서 관리하는게 유지보수에 용이합니다.
- setState function으로 state값을 업데이트 시켜주므로 해당 state변수는 return 시키지 않아도 됩니다.

Reply:
와우 설명 진짜 감사합니다 하나 깨닫고 갑니다

Reply:
컴포넌트를 let이 아닌 const로 선언해야 하는 이유?
변하는 값이기 때문에 let으로 선언해야 할 것 같지만, 원시 데이터(state)를 건드리지 않고 불변성을 지키기 위해서는 const로 선언해야 한다.

Reply:
is라는 naming은 boolean naming convention으로서, 일반 변수명에의 사용은 적합하지 않음!
```suggestion
  const [id, setId] = useState('');
  const [pw, setPw] = useState('');
```